pkgs/misc/vscode-registries/openvsx/nix-prefetch-openvsx/default.nix

```
@@ -0,0 +1,69 @@
+{ stdenvNoCC
+, lib
+, makeWrapper
+, shellcheck
+, coreutils
+, curl
+, jq
+, unzip
+, nix
+, doCheck ? true
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "nix-prefetch-openvsx";
+  version = "0.2.0";
+
+  src = lib.cleanSourceWith {
+    src = ./.;
+    filter = name: type:
+      lib.cleanSourceFilter name type
+      && ! lib.hasSuffix ".nix" name
+      && name != "flake.lock";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    coreutils
+    curl
+    jq
+    unzip
+    nix
+  ];
+
+  inherit doCheck;
+  checkInputs = [ shellcheck ];
+
+  checkPhase = ''
+    runHook preCheck
+    shellcheck "$src/${pname}"
```

Please check the file in out.

```suggestion
    shellcheck "$out"
```